pyasn1/codec/ber/decoder.py

```diff
@@ -5,15 +5,14 @@
 # License: http://snmplabs.com/pyasn1/license.html
 #
 import os
-import sys
-from io import BytesIO, BufferedReader, IOBase, DEFAULT_BUFFER_SIZE
 
 from pyasn1 import debug
 from pyasn1 import error
+from pyasn1.codec import streaming
```

Comment:
```suggestion
from pyasn1.codec.streaming import read, isEndOfStream, asSeekableStream
```
I suggest to import these objects and use them directly (as shown below, every module member lookup adds ~20 ns), basically anywhere where `straming.[...]` is used now.

Reply:
Done